core/src/main/scala/org/bitcoins/core/util/Wrappers.scala

```diff
@@ -0,0 +1,19 @@
+package org.bitcoins.core.util
+
+trait SeqWrapper[+T] extends Seq[T] {
+  def wrapped: Seq[T]
```

Comment:
I think this is confusing, as you are technically _unwrapping_ the type the extends this and exposing the underlying implementation type.